src/components/encounter/encounter-form.component.tsx

```diff
@@ -611,6 +611,23 @@ const EncounterForm: React.FC<EncounterFormProps> = ({
       value = value ? ConceptTrue : ConceptFalse;
     }
 
+    if (codedTypes.includes(field.questionOptions.rendering)) {
+      field.questionOptions.answers.forEach((answer) => {
+        if (answer.disable?.disableWhenExpression) {
```

Comment:
Can we do a soft check of whether "this" field ID was referenced in the expression?

```ts
const disableExpression = answer.disable?.disableWhenExpression;

if (disableExpression?.includes("myValue") || disableExpression?.includes(field.id)) {
    // evaluate
}

```

Reply:
Added a check for `myValue`, since if the expression includes field id, it will register itself as it's own dependant and trigger the expression evaluation as a dependant.